src/types/plugin-inputs.ts

```diff
@@ -20,6 +20,7 @@ export const pluginSettingsSchema = T.Object({
     .Encode((value) => value.toString()),
   shouldUseGithubStorage: T.Boolean({ default: false }),
   storageOwner: T.String({ default: "ubiquity-os-marketplace" }),
+  maxCompletionTokens: T.Number({ default: 7000 }),
```

Comment:
Why 7000

Reply:
`/ask` defaults to 10k, it was 8k before and telegram with huge blocks of texts is uneasy on the eyes at times

https://github.com/ubiquity-os-marketplace/command-ask/blob/782c838eae61d977133d4b071987cb213ab6182a/src/types/plugin-inputs.ts#L26

Reply:
I think if you don't put anything it defaults to max which makes more sense to use.

Reply:
cool will do.

Can you price this task for me?

And is there any chance you could message me on TG?